UPGRADE.md

```diff
@@ -56,6 +56,12 @@ if ('contao_backend' === $route) {
 }
 ```
 
+Use the `contao.csrf.token_manager` service instead of `REQUEST_TOKEN`:
```

Comment:
Should we also hint that you can use the global template variables?

Reply:
And maybe, that `$GLOBALS['TL_CONFIG']['disableRefererCheck']` was removed? :see_no_evil:

Reply:
Added hint for global template variables.
Not sure about those config settings. 
@contao/developers WDYT?

Reply:
Added in 23b0d4834ab833c1b4fd626169e6841f58317f57.